examples/webpack/example-builder.js

```diff
@@ -282,9 +281,12 @@ export default class ExampleBuilder {
     return (
       source
         // remove "../src/" prefix to have the same import syntax as the documentation
-        .replace(/'\.\.\/src\//g, "'")
+        .replaceAll(
+          /(["'])(\.\.\/src\/ol\/[^"']+\.js)\1/g,
+          (full, quote, path) => "'" + path.slice(7) + "'",
+        )
```

Comment:
With this updated regexp both single and double quotes will work, it outputs only single quote paths.
I thought I'd change the few double quoted paths for consistency, but it is not required to fix this.

Single quotes could be enforced in examples by throwing and exception here if `quote !== "'"`.

Reply:
I don't think we'll want to throw an exception here. All linting should be done with eslint for consistency.

But now I understand the goal of this pull request. Thanks for the explanation, @MoonE. So I think you can merge this, but going forward I think `@type` annotations should be removed instead of changing their module paths.